Application/components/Server/MemberList.tsx

```diff
@@ -23,15 +21,22 @@ export function MemberList({isAdmin, chatID, isView}: any) {
   const [searchResult, setSearchResult] = useState<number | null>(null);
   const [isAdmin1, setIsADmin] = useState(isAdmin);
   const { user } = useUser();
+  const { activeView, setActiveView } = useChat();
+  const [myID, setMyID] = useState<string>('');
+
+  useEffect(() => {
+    if (user && user.id) {
+      // Set sender to user's username if user exists and username is not null/undefined
+      setMyID(user.id);
+    }
+  }, [user]); // Dependency array ensures this runs whenever `user` changes
 
   const removeMember = async(member: String, index: any) =>{
-      console.log(membersIDList[index] + " ahahahahahah")
       const memberToRemove = membersIDList[index];
       let membersIDListToSort = membersIDList.filter(item => item !== memberToRemove);
       membersIDListToSort.sort();
       const rawChannelKey = `chat:${membersIDListToSort.join(",")}`;
-      const newChannelKey = generateHash(rawChannelKey);
-      // alert("Something")
+      const newChannelKey = generateHashFromString(rawChannelKey);
```

Comment:
- The code uses generateHashFromString from @/utility to generate a channel key by concatenating member IDs. However, without knowledge of the hashing algorithm used in generateHashFromString, it's unclear if it provides sufficient security for generating channel keys. It's recommended to use a well-established cryptographic hash function like SHA-256 for secure hashing.

Reply:
My guy, `generateHashFromString` is a function defined in `utility` that uses `SHA-256`.